src/mlpack/methods/ann/ffn.hpp

```diff
@@ -142,6 +142,16 @@ class FFN
   void Predict(arma::mat& predictors, arma::mat& results);
 
   /**
+   * Predict the responses to a given set of predictors. The responses will
+   * reflect the output of the given output layer as returned by the
+   * output layer function.
+   *
+   * @param predictors Input predictors.
+   * @return Output predictions of responses into.
+   */
+  arma::mat Predict(arma::mat predictors);
+
```

Comment:
Ah, ok.  I missed your comment at 12:38pm, I must have been typing mine out then.

I agree with @zoq here---there is no other `Predict()` function in the library that operates like `mat Predict(const mat&)`, so I think it is better to be consistent and only provide the top two overloads.  If we eventually want to switch all `Predict()` and similar calls to `mat Predict()`, we can discuss that, but the place to discuss that would be some other issue I think (and it would also probably be a huge amount of work).